### PR TITLE
Use single Couchbase bucket

### PR DESCRIPTION
**Pull request by stingion**

### CODE REVIEW

**Review by PavloEsco — DISMISSED** [review]
Tested Ok on the local environment

**Review by PavloEsco — DISMISSED** [review]
Tested OK

**Review by dmytro-lazebnyi — DISMISSED** [review]

File: scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java [hunk 1]
```diff
@@ -82,6 +82,15 @@ private void startCouchbase() {
             .password(password)
             .roles(Collections.singletonList(new UserRole("bucket_full_access", name))),
         true);
+
+    String configName = "configurations";
+    couchbase.createBucket(
+        DefaultBucketSettings.builder().name(configName).password(password).quota(100).build(),
```
Comment (aharonha) [inline scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java]: why did you use `quota(100)`?

File: scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/SaveRequest.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
   private Object token;
   private String repository;
   private String key;
-  private JsonNode value;
+  private Object value;
```
Comment (aharonha) [inline scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/SaveRequest.java:11]: Why did you change to `Object`?

File: scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java [hunk 3]
```diff
@@ -82,6 +82,15 @@ private void startCouchbase() {
...
+        UserSettings.build()
+            .name(configName)
+            .password(password)
+            .roles(Collections.singletonList(new UserRole("bucket_full_access", configName))),
```
Comment (segabriel) [inline scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java]: make it as a constant value `bucket_full_access`

File: scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java (hunk 1 above)
Comment (dmytro-lazebnyi) [reply scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java]: Minimally allowed bucket quota. 
Actually, this is a redundant code here since `DefaultBucketSettings` sets `100` as the default value.

File: scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/SaveRequest.java (hunk 2 above)
Comment (dmytro-lazebnyi) [reply scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/SaveRequest.java:11]: To decouple public API from `jackson`.

File: scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java (hunk 1 above)
Comment (segabriel) [reply scalecube-configuration-benchmarks/src/main/java/io/scalecube/configuration/benchmarks/Environment.java]: So can we remove such settings from this statement?

**Review by ronenhamias — DISMISSED** [review]

**Review by dmytro-lazebnyi — APPROVED** [review]

**Review by segabriel — APPROVED** [review]
